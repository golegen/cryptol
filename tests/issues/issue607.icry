let (x,y)=(0,x+1)
x
y
