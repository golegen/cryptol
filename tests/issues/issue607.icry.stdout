Loading module Cryptol
[warning] at <interactive>:1:5--1:18:
  Defaulting the type of '<interactive>::y' to [1]
0x0
0x1
